deployments/compose/docker-compose.yml

```diff
@@ -182,7 +182,12 @@ services:
 
   spark-thrift:
     image: docker.io/bitnami/spark:3
-    command: ["bash","-c", "/opt/bitnami/entrypoint.sh"]
+    # Windows bug (see https://github.com/treeverse/lakeFS/pull/2871)
+    # Need to copy the mounted file and replace the line endings for Windows hosts.blkio_config:
+    # For linux machines, this is a no-op
+    command: ["bash","-c", "cp /opt/bitnami/entrypoint.sh /opt/bitnami/entrypoint-copy.sh &&
```

Comment:
prefer not to enable this hack - I assume that volume mount should work if the file format is kept

Reply:
Would it work if we created a new Dockerfile that started with docker.io/bitnami/spark:3 and edited that file, creating a new good Docker image?